server/test/utils.ts

```diff
@@ -60,11 +49,16 @@ async function createUser({
     github_username: github_username ?? randomEmptyChance(20, randUserName()),
     website: website ?? randomEmptyChance(20, randUrl()),
   };
+}
 
+async function createUser(options: Partial<UserObject> = {}) {
+  const userObject = makeUserObject(options);
   // insert into DB after creation
-  const user = await User.create(userObject);
+  return await User.create(userObject);
+}
 
-  return user;
+function createManyUsers(numberOfUsers: number): Partial<UserObject>[] {
```

Comment:
Given the current naming, this `makeUserObjects` (plural form of `makeUserObject`) as it's not inserting the users.